create-i18n/index.d.ts

```diff
@@ -15,7 +15,7 @@ export interface ComponentsJSON {
 
 export interface TranslationFunction<
   Arguments extends any[] = any[],
-  Output extends TranslationJSON = TranslationJSON
+  Output = TranslationJSON | Translation
```

Comment:
added `Translation` here to be able to have a combination of `count` and `params`